src/graphql/resolvers.js

```diff
@@ -174,6 +174,43 @@ module.exports = function Resolvers (ssb) {
     })
   }
 
+  /**
+   * Gets all the public threads that room members have posted or replied to
+   * @param {object} opts - optional parameters
+   * @param {int} [opts.limit=10] - max amount of threads to return
+   * @param {int} opts.threadMaxSize - max amount of messages in each thread to return
+   */
+  const getPublicThreads = (opts) => {
+    const { threadMaxSize, limit = 10 } = opts
+    const members = ssb.room.members()
+  
+    return new Promise((resolve, reject) => {
+      pull(
+        ssb.threads.public({ threadMaxSize, allowlist: ['post'], following: true }),
+        // hacky! Only threads that contain a message from a member of the room
```

Comment:
why would the graphql server be following people not in the room?
I think what you're doing with filter looks fine, but I would say "this could be more optimised" if that's why it's hacky